src/schema/v2/me/manageArtworksCollectionsMutation.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  GraphQLObjectType,
+  GraphQLUnionType,
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLList,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { ResolverContext } from "types/graphql"
+import { CollectionType } from "./collection"
+import numeral from "../fields/numeral"
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "ManageArtworksCollectionsSuccess",
+  isTypeOf: (data) => data.counts,
+  fields: () => ({
+    counts: {
+      type: new GraphQLObjectType<any, ResolverContext>({
+        name: "ManageArtworksCollectionsCounts",
+        fields: {
+          artworks: numeral(({ artworks }) => artworks),
+          addedToCollections: numeral(({ added_to }) => added_to),
+          removedFromCollections: numeral(({ removed_from }) => removed_from),
+        },
+      }),
+      resolve: (response) => {
+        return response.counts
+      },
+    },
+    addedToCollections: {
+      type: new GraphQLList(CollectionType),
+      resolve: (response) => {
+        return response.added_to
+      },
+    },
+    removedFromCollections: {
+      type: new GraphQLList(CollectionType),
+      resolve: (response) => {
+        return response.removed_from
+      },
+    },
+  }),
+})
+
+const ErrorType = new GraphQLObjectType<any, ResolverContext>({
+  name: "ManageArtworksCollectionsFailure",
+  isTypeOf: (data) => {
+    return data._type === "GravityMutationError"
+  },
+  fields: () => ({
+    mutationError: {
+      type: GravityMutationErrorType,
+      resolve: (err) => (typeof err.message === "object" ? err.message : err),
+    },
+  }),
+})
+
+const ResponseOrErrorType = new GraphQLUnionType({
+  name: "ManageArtworksCollectionsResponseOrError",
+  types: [SuccessType, ErrorType],
+})
+
+interface InputProps {
+  artworkIDs: string[]
+  addToCollectionIDs: string[]
+  removeFromCollectionIDs: string[]
+}
+
+export const manageArtworksCollectionsMutation = mutationWithClientMutationId<
+  InputProps,
+  any,
+  ResolverContext
+>({
+  name: "manageArtworksCollections",
```

Comment:
**suggestion:** Mutation names are typically capitalized (that would fix the caps for the `input` type as well)